core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4091,14 +4095,14 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String USER_BLOCK_WORKER_CLIENT_READ_RETRY =
         "alluxio.user.block.worker.client.read.retry";
     public static final String USER_BLOCK_WRITE_LOCATION_POLICY =
-        "alluxio.user.block.write.location.policy.class";
+        "alluxio.user.block.write.location.policy";
     public static final String USER_CONF_CLUSTER_DEFAULT_ENABLED =
         "alluxio.user.conf.cluster.default.enabled";
     public static final String USER_CONF_SYNC_INTERVAL = "alluxio.user.conf.sync.interval";
     public static final String USER_DATE_FORMAT_PATTERN = "alluxio.user.date.format.pattern";
     public static final String USER_FILE_BUFFER_BYTES = "alluxio.user.file.buffer.bytes";
-    public static final String USER_FILE_COPY_FROM_LOCAL_BLOCK_LOCATION_POLICY =
-        "alluxio.user.file.copyfromlocal.block.location.policy.class";
+    public static final String USER_FILE_COPYFROMLOCAL_BLOCK_LOCATION_POLICY =
+        "alluxio.user.file.copyfromlocal.block.location.policy";
```

Comment:
This is new in 2.0 right? Can we change this to be `*.copy.from.local.*`? We should follow paradigm of the rest of the property keys and separate words by `.`

Reply:
Also would like to add that I like using `.class` as well as keeping words in the var name separated by `_`

Reply:
mark as deprecated , create an issue to fix local first.